### PR TITLE
Auto packaging

### PR DESCRIPTION
**Pull request by kumagi**
Automatically building .deb files workflow is very very useful!

### CODE REVIEW

**Comment by kumagi** [comment]
I cannot test releasing workflow. Is there any good way to do it?

File: packaging/debian/control [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ With development files, you can build your own SXG applications.
+
+Package: sxg-util
+Section: web
+Architecture: any
+Multi-Arch: foreign
+Depends: libsxg0.2 (= ${binary:Version}),
+         ${misc:Depends},
+         ${shlibs:Depends}
+Description: Signed HTTP Exchange utilities to manipurate SXG files
```
Comment (banaag) [inline packaging/debian/control:45]: manipurate -> manipulate

**Review by banaag — APPROVED** [review]

File: src/sxg_cert_chain.c [hunk 2]
```diff
@@ -161,6 +161,7 @@ bool sxg_execute_ocsp_request(BIO* io, const char* path, OCSP_CERTID* id,
       case 0:
         success = false;
       case 1:
+        break;
```
Comment (twifkak) [inline src/sxg_cert_chain.c:164]: Oh, good catch.

**Comment by twifkak** [comment]
Wow, thanks for this change!

I'm not familiar with GH workflows; I don't know how to test them. Perhaps you could try something like https://github.com/nektos/act?

File: packaging/debian/control (hunk 1 above)
Comment (kumagi) [reply packaging/debian/control:45]: Thanks!

**Comment by kumagi** [comment]
@twifkak I don't know how to test the workflow and https://github.com/nektos/act will run in Docker to run my packaging process with Docker. It would be helpful for workflow development but does not guarantee the same behavior in CI. 
As a first-aid to this problem, I put two almost similar workflow one will return success or not for building deb file and  other is to build&upload deb file to the Github repository. As long as these two workflow is almost the same, failure of first one will detect failure of second one.

**Pull request merged**